en/SUMMARY.md

```diff
@@ -339,18 +292,67 @@
+  * [Data Links](data_links/README.md)
+    * [FrSky Telemetry](peripherals/frsky_telemetry.md)
+    * [Satellite Coms (Iridium/RockBlock)](advanced_features/satcom_roadblock.md)
+    * [MAVLink Telemetry (OSD/GCS)](peripherals/mavlink_peripherals.md)
+    * [Telemetry Radios](telemetry/README.md)
+      * [SiK Radio](telemetry/sik_radio.md)
+        * [RFD900 (SiK) Telemetry Radio](telemetry/rfd900_telemetry.md)
+        * [HolyBro (SIK) Telemetry Radio](telemetry/holybro_sik_radio.md)
+      * [Telemetry Wifi](telemetry/telemetry_wifi.md)
+        * [ESP8266 WiFi Module](telemetry/esp8266_wifi_module.md)
+        * [ESP32 WiFi Module](telemetry/esp32_wifi_module.md)
+        * [3DR Telemetry Wifi (Discontinued)](telemetry/3dr_telemetry_wifi.md)
+      * [Microhard Serial Telemetry Radio](telemetry/microhard_serial.md)
+        * [ARK Electron Microhard Serial Telemetry Radio](telemetry/ark_microhard_serial.md)
+        * [Holybro Microhard P900 Telemetry Radio](telemetry/holybro_microhard_p900_radio.md)
+      * [CUAV P8 Telemetry Radio](telemetry/cuav_p8_radio.md)
+      * [HolyBro XBP9X - Discontinued](telemetry/holybro_xbp9x_radio.md)
+    * [TBS Crossfire (CRSF) Telemetry](telemetry/crsf_telemetry.md)
+  * [Power Modules/PDB](power_module/README.md)
+    * [CUAV HV pm](power_module/cuav_hv_pm.md)
+    * [CUAV CAN PMU](dronecan/cuav_can_pmu.md)
+    * [Holybro PM02](power_module/holybro_pm02.md)
+    * [Holybro PM07](power_module/holybro_pm07_pixhawk4_power_module.md)
+    * [Holybro PM06 V2](power_module/holybro_pm06_pixhawk4mini_power_module.md)
+    * [Holybro PM02D (digital)](power_module/holybro_pm02d.md)
+    * [Holybro PM03D (digital)](power_module/holybro_pm03d.md)
+    * [Pomegranate Systems Power Module](dronecan/pomegranate_systems_pm.md)
+    * [Sky-Drones SmartAP PDB](power_module/sky-drones_smartap-pdb.md)
+  * [Smart Batteries](smart_batteries/README.md)
+    * [Rotoye Batmon Battery Smartification Kit](smart_batteries/rotoye_batmon.md)
+  * [Peripherals](peripherals/README.md)
+    * [ADSB/FLARM (Traffic Avoidance)](peripherals/adsb_flarm.md)
+    * [Air Traffic Avoidance: ADSB/FLARM](advanced_features/traffic_avoidance_adsb.md)
+    * [Air Traffic Avoidance: UTM](advanced_features/traffic_avoidance_utm.md)
+    * [Camera](peripherals/camera.md)
+    * [Gimbal \(Mount\) Configuration](advanced/gimbal_control.md)
+    * [Grippers](peripherals/gripper.md)
+      * [Servo Gripper](peripherals/gripper_servo.md)
+    * [Parachute](peripherals/parachute.md)
     * [I2C Peripherals](sensor_bus/i2c_general.md)
       * [I2C bus accelerators](sensor_bus/i2c_general.md#i2c-bus-accelerators)
       * [TFI2CADT01 I2C address translator](sensor_bus/translator_tfi2cadt.md)
     * [CAN Peripherals](can/README.md)
     * [DroneCAN Peripherals](dronecan/README.md)
       * [PX4 DroneCAN Firmware](dronecan/px4_cannode_fw.md)
       * [ARK CANnode](dronecan/ark_cannode.md)
+    * [Remote ID](peripherals/remote_id.md)
```

Comment:
@sfuhrer You had this in operations. From a pilot point of view it is invisible - you need to know about it when assembling a drone, so put it here.

Reply:
Okay, putting it in either is probably fine, and interlink it.

Reply:
Or do you split up the "integration" from the "operational" parts?

Reply:
I am still getting my head around what operational means. Where would integration live - i.e. the basic vehicle assembly guide?

Reply:
@sfuhrer What I'd like to do is have a guide on the integration/assembly on a per-vehicle basis.

This would be like a wiring quickstart but unbranded and also vehicle specific. It would be centred around a diagram like the wiring chart in the assembly guide - like this one: https://docs.px4.io/main/en/assembly/quick_start_pixhawk6x.html#wiring-chart-overview and have similar instructions on writing up the bits an bobs.

So this would be the image for an MC, but we'd probably have more appropriate outputs for a Quadplane VTOL (say).
The instructions would be assuming Pixhawk standard outputs and point to the hardware section for options.

These existing specific assembly guides would go below the flight controller docs.

My reasoning is that even though there is a lot of common information for assembly there are lots of things that aren't well covered. For example, we talk very little about what outputs you might need for plane and VTOL anywhere other than to say you need a BEC.

I'd be OK for the guide to be common, but only if we can expand out the powering and wiring for the other frames. Either way though, I think it would help to present that as part of the instructions for each vehicle.

Ultimately I'd prefer that users face this almost entirely through vehicle-specific docs.

Make sense?

Reply:
Yes. Though I was thinking we might duplicate even more - and have a per vehicle type version of  https://docs.px4.io/main/en/assembly/quick_start_pixhawk6x.html#wiring-chart-overview with links to setup for the bits that are per-vehicle.

I won't know until we get to that point and I try some alternatives. But ideally we're trading off as vehicle specific as possible with some duplication.